pkgs/development/node-packages/node-packages.json

```diff
@@ -105,6 +105,7 @@
 , "graphql-cli"
 , "graphql-language-service-cli"
 , "grunt-cli"
+, "gsts"
```

Comment:
Please use `buildNpmPackage` instead. See #229475.

Reply:
Thanks for the comment @dotlambda.
Would you happen to know how to address the error in https://github.com/NixOS/nixpkgs/pull/344604#issuecomment-2376653731?
🙏

Reply:
That sounds like a broken lock file. I suggest regenerating `package-lock.json` and if that helps opening an upstream PR.